src/omv/py/py_image.c

```diff
@@ -7594,6 +7625,8 @@ static const mp_rom_map_elem_t globals_dict_table[] = {
     {MP_ROM_QSTR(MP_QSTR_CODE93),              MP_ROM_INT(BARCODE_CODE93)},
     {MP_ROM_QSTR(MP_QSTR_CODE128),             MP_ROM_INT(BARCODE_CODE128)},
 #endif
+    {MP_ROM_QSTR(MP_QSTR_INTERPOLATE_BILINEAR),MP_ROM_INT(INTERPOLATE_BILINEAR)},
+    {MP_ROM_QSTR(MP_QSTR_IMAGE_CENTER),        MP_ROM_INT(IMAGE_CENTER)},
```

Comment:
Can you rename these with a PREFIX. Like IMAGE_HINT_BILINEAR, IMAGE_HINT_CENTER.